### PR TITLE
repacle embedding in fluid with 2.0 version

### PR DESCRIPTION
**Pull request by Difers**
### PR types

Others 
### PR changes

APIs 
### Description

To clean the code in paddle.fluid, the useless codes should be removed and code still be used should move to a new position.

For function paddle.fluid.layers.nn.embedding, since it is not used in version 2.x, should be removed.

### CODE REVIEW

File: test/legacy_test/test_communicator_geo.py [hunk 1]
```diff
@@ -36,9 +36,9 @@ def net(self):
             name='x1', shape=[-1, 1], dtype='int64', lod_level=1
         )
 
-        emb = fluid.layers.embedding(
+        emb = paddle.static.nn.embedding(
             input=x1,
-            size=[10000, 10],
+            size=[10000, 13],
```
Comment (zoooo0820) [inline test/legacy_test/test_communicator_geo.py]: 这里改动size的原因是？

**Review by jeff41404 — APPROVED** [review]
LGTM

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**